app/services/teacher_training_api/import_course.rb

```diff
@@ -19,7 +19,7 @@ def initialize(provider:, course:)
     end
 
     def call
-      return unless IMPORTABLE_STATES.include?(attrs[:state])
+      return unless IMPORTABLE_STATES.include?(attrs[:state]) && !further_education_level_course?
```

Comment:
The double negation (I think that's the right computer words) makes this hard to read. Would this be better? 

```suggestion
      return unless IMPORTABLE_STATES.include?(attrs[:state])
      return if further_education_level_course?
```